notebooks/17-framework-usage.ipynb

```diff
@@ -0,0 +1,506 @@
+{
```

Comment:
<p>It seems DetectorService is pretty dependant on FeatureManager. Maybe we can instantiate the FeatureManager inside the <strong>init</strong> of DetectorService? I think it would be less redundant, and we would still be able to access it (feature_manager = detector_service.feature_manager)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bleu/farcaster-social-graph/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
agree! that makes it more composable and easier to run experiments in notebooks